docs/team/siqiang-ng.md

```diff
@@ -3,79 +3,67 @@ layout: page
 title: Ng Si Qiang's Project Portfolio Page
 ---
 
-## Project: Projact
-
-Projact is a **project management app that helps NUS computing students to organise their fellow computing students' contacts and their teams' meeting links and tasks**. It is optimized for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI).
+## Project: Projact 
```

Comment:
do you want to make it ### to save more space? but if it already fits then should be fine!

Reply:
It already fits haha. Previously I put ### but the horizontal line is gone and the space is gone between the first two para